src/pages/CropSidebar/CropSidebar.js

```diff
@@ -323,6 +323,7 @@ const CropSidebar = ({
 
   const updateRegionRedux = (regionName) => {
     const selectedRegion = regionsRedux.filter((region) => region.shorthand === regionName)[0];
+    localStorage.setItem('region', selectedRegion.id);
```

Comment:
this needs to happen in landing as well. it's always using the default region id of 59 and returning the same dictionary